server/db/models/product.js

```diff
@@ -0,0 +1,26 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Product = db.define('product', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  description: {
+    type: Sequelize.TEXT,
+    allowNull: false
+  },
+  price: {
+    type: Sequelize.DECIMAL(10,2),
+    allowNull: false
+  },
+  imageUrl: {
+    type: Sequelize.STRING,
+    defaultValue: 'http://www.indiantradebird.in/images/noimage.jpg',
+    validate: {
+      isUrl: true
```

Comment:
@garrett-green I really like this validation, are we going to use the files that we have in our public directory? and if that the case do you know if that will allow the path without the http://www?

Reply:
@tungsteno22 When we create our final seed file with _official_ dummy data we'll use an image in our public directory but until then, this URL will serve as the default.